plugins/module_utils/_crypto/module_backends/csr.py

```diff
@@ -906,6 +1001,21 @@ def get_csr_argument_spec() -> ArgumentSpec:
                 "mutually_exclusive": [("full_name", "relative_name")],
                 "required_one_of": [("full_name", "relative_name", "crl_issuer")],
             },
+            "custom_extensions": {
+                "type": "list",
+                "elements": "dict",
+                "options": {
+                    "critical": {"type": "bool", "default": False},
+                    "oid": {"type": "str"},
```

Comment:
Shouldn't the `oid` be required?
```suggestion
                    "oid": {"type": "str", "required": True},
```